### DISCUSSION TITLE
Focus management on Android TV

### DISCUSSION THREAD

**Discussion by rlalance** [dotnet/maui#10125] [Q&A]
Hi,

I was wondering if someone could suggest a nuget package to assist with implement focus management on android TV.
I am on nvidia shield and the application runs fine, but, none of the input on the remote works for moving focus.

**Comment by AbanoubZak** [comment]
Apply focus() on one of the elements on start and make sure you have a supported layout, for example, don't use a menu use a regular content page with buttons.
It worked for me.
